lib/AuthStateManager.ts

```diff
@@ -136,7 +137,7 @@ export class AuthStateManager {
       };
 
       this._sdk.tokenManager.getTokens()
-        .then(({ accessToken, idToken }) => {
+        .then(({ accessToken, idToken, refreshToken }) => {
```

Comment:
As discussed, should we hide the refreshToken from authState?

Reply:
I'm not sure we should entirely HIDE it - so much as let people not have to worry about it.